libdevcrypto/Common.cpp

```diff
@@ -259,12 +280,12 @@ h256 Nonce::get(bool _commit)
 			BOOST_THROW_EXCEPTION(InvalidState());
 		
 		// prevent seed reuse if process terminates abnormally
-		writeFile(s_seedFile, bytes());
+		try { writeFile(s_seedFile, bytes()); } catch (FileError const&) {}
```

Comment:
As per comment on previous line, this needs to throw. If it crashes: perfect! There's no sense or use in hiding something like this.